OracleTuxedo/art_runtime/dockerfiles/12.2.2/Dockerfile.template

```diff
@@ -0,0 +1,84 @@
+#
+# Dockerfile template for Tuxedo ART 12.2.2
+# 
+# For Tuxedo 12.2.2, you need JRE installed first, so before you can build this image, you must download the Oracle Server JRE binary and drop in folder OracleJDK/java-8 and build that image.
+#   cd OracleJDK/java-8
+#   sh build.sh
+
+# Pull base image
+FROM oracle/tuxedo:12.2.2
+
+MAINTAINER Judy Liu <judy.liu@oracle.com>
+
+# Create the installation directory tree and user oracle with a password of oracle
+USER root
+RUN yum -y install perl perl-CPAN ksh mksh sudo make vim tar net-tools dos2unix tk tcl-devel tk-devel gtk2 expect libXtst gnome-desktop xorg-x11-xauth dejavu-lgc-sans-fonts liberation-sans-fonts libcanberra-gtk2 PackageKit-gtk3-module libaio openssh-server openssh-client rsync; yum -y clean all 
```

Comment:
Why are you installing a Gnome Desktop, X11 applications and fonts? These sorts of things don't really belong inside a container.

Also, we don't allow an SSH daemon to be run inside a container so please remove `openssh-server` from this list.

Finally, please replace the `;` with `&&` .